db/re/skill_cast_db.txt

```diff
@@ -879,15 +879,15 @@
 
 //===== Ninja ==============================
 //-- NJ_KUNAI
-524,0,1000,0,0,0,0,0
+524,0,0,0,0,0,200,0
 //-- NJ_HUUMA
-525,2400,2000,0,100,0,0,600
+525,1000,2000,0,100,0,0,500
```

Comment:
The after-cast delay on kRO is 500ms instead of 2000ms for Throw Huuma Shuriken.